functime/plotting.py

```diff
@@ -115,7 +122,7 @@ def plot_forecasts(
     """
 
     # Get most recent observations
-    entity_col, time_col, target_col = y_true.columns
+    entity_col, time_col, target_col = y_true.columns[:2]
```

Comment:
```suggestion
    entity_col, time_col, target_col = y_true.columns[:3]
```

Reply:
oh yes you are right, :3 takes cols 0 1 and 2.